lib/msf/core/exploit/smb/shadow_mitm_dispatcher.rb

```diff
@@ -2,7 +2,9 @@ module Msf
 class Exploit
 module SMB
   # This class provides a Man-In-The-Middle packet Dispatcher.
-  class MitmDispatcher < RubySMB::Dispatcher::Base
+  # The dispatcher only supports SMB2 for now.
+  # TODO: SMB3 only requires signing on the TreeConnectRequest.
+  class ShadowMitmDispatcher < RubySMB::Dispatcher::Base
```

Comment:
Thanks! :+1:

Reply:
Using `ShadowSMB2MitmDispatcher` with Zietwerk would require the file name to be `shadow_s_m_b_2_mitm_dispatcher.rb`.
I decided to remove the `SMB2` part as we can add support for other smb versions in the future.